### DISCUSSION TITLE
다큐멘테이션 및 언어 어떻게 하죠?

### DISCUSSION THREAD

**Discussion by Namhoon-Cho** [Looking-4-Attention/AttentionMe#4] [Ideas]
저희 다큐멘테이션 어떻게 할까요? 기술 문서들 필요해보이는데. 수업 시간에 한 것 따라할까요?
그리고 저희 커밋 메시지 같은거 꼭 영어로 써야 하나요?

**Comment by iamdami** [comment]
기술 문서들 수업 시간에 한 것처럼 해도 좋을 거 같아요!
커밋 메시지는 한국어로 써도 될 거 같아요
Readme는 한/영 병기하는 거 어때요?

**Reply by Namhoon-Cho** [reply]
오 좋아요 그대로 진행해도 되겠는데요?

**Reply by hyun1227** [reply]
좋아요!

**Reply by wwwwje2008** [reply]
좋네용

**Comment by Namhoon-Cho** [comment]
여러분 한영표기는 리드미만 하고 한국어 우선주의 아니었나요,,, 디스커션을 더 꼼꼼히 확정지어야 한다는 배움,,, 아직 클로즈된 디스커션 아니라지만,,,다들 코드 내부에서 메시지 보낼 때 영어 쓰셨군요,,, 메인테이너님,,,제 것도 그냥 영어로 통일해주십시오,,,
